docs/DeveloperGuide.md

```diff
@@ -2,33 +2,82 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Class diagram for v2.0
+![Class Diagram](assets/classdiagram.svg)
+
+We have 4 packages in total: Control, Entity, Seedu.Duke(boundary) and exception. 
```

Comment:
Good brief description of the Class diagram. Maybe you can do the same thing for the Sequence Diagram.